finance/models.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from candidate.models import SocialMediaModel, Person
+
+
+@python_2_unicode_compatible
+class Committee(SocialMediaModel):
+    COMMITTEE_TYPES = (
+        ('OF', 'Primarily-formed Official Committee'),
+        ('PF', 'Primariliy-formed Committee'),
+        ('IC', 'Independent Committee')
+    )
+    name = models.CharField(max_length=255)
+    address = models.ForeignKey('locality.Address', null=True)
+    locality = models.ForeignKey('locality.Locality', null=True)
+    committee_id = models.CharField(max_length=16, unique=True)
```

Comment:
I might name this something different than committee_id. I'm assuming this Id is coming from a third-party (netfile or something). But I also see us using the name `committee_id` on other models to refer to Committee.id, which might be confusing.

Is there another name the source has given it? Filer id?